### PR TITLE
set default value of outputDir to '.', fixes #54

### LINKED ISSUE #54: Set default value for outputDir
I just upgraded karma-junit-reporter from v0.2.2 to v0.3.3.  Test runs fail with error: "You must set an output directory for JUnitReporter via the outputDir config property".  It seems that this setting had a default value before, but is now required.  Can we please revert to the default value?  It was working just fine.  Thanks.

### PR DESCRIPTION
**Pull request by janeklb**
Fixing #54

### CODE REVIEW

**Comment by janeklb** [comment]
I signed it!

File: index.js [hunk 1]
```diff
@@ -8,11 +8,7 @@ var JUnitReporter = function (baseReporterDecorator, config, logger, helper, for
   var reporterConfig = config.junitReporter || {}
   var pkgName = reporterConfig.suite || ''
 
-  if (!reporterConfig.outputDir) {
-    throw new Error('You must set an output directory for JUnitReporter via the outputDir config property')
-  }
-
-  var outputDir = helper.normalizeWinPath(path.resolve(config.basePath, reporterConfig.outputDir)) + path.sep
+  var outputDir = helper.normalizeWinPath(path.resolve(config.basePath, reporterConfig.outputDir || '.')) + path.sep
```
Comment (Dignifiedquire) [inline index.js]: Let's use a proper check against `null` and `undefined`, sth like this.

```js
var configOutDir = '.'
if (reporterConfig.outputDir != null) {
  configOutDir = reporterConfig.outputDir
}
```

**Comment by Dignifiedquire** [comment]
Thanks, can you please change the commit message to follow our [guidelines](http://karma-runner.github.io/0.13/dev/git-commit-msg.html)

**Comment by janeklb** [comment]
hope you don't mind the amended commit - how's it look now?

**Comment by Dignifiedquire** [comment]
Thanks a lot :octocat:

**Comment by Dignifiedquire** [comment]
Released in `0.3.4`

**Pull request merged**